Makefile.am

```diff
@@ -2042,6 +2042,12 @@ write_riemann_la_CFLAGS = $(AM_CFLAGS) $(LIBRIEMANN_CLIENT_CFLAGS)
 write_riemann_la_LDFLAGS = $(PLUGIN_LDFLAGS) $(LIBRIEMANN_CLIENT_LIBS)
 endif
 
+if BUILD_PLUGIN_WRITE_SYSLOG
```

Comment:
Please keep this list sorted, this should come after the BUILD_PLUGIN_WRITE_SENSU section

Reply:
fixed

Reply:
and after write_stackdriver, the t comes before the y.

Reply:
fixed